src/Microsoft.AspNetCore.Mvc.Razor.Extensions/baseline.net45.json

```diff
@@ -0,0 +1,1689 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.Mvc.Razor.Extensions, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
We don't need these baselines files. There is no compatibility bar between this and the old assembly.